packages/ui/Scrollable/Scrollable.js

```diff
@@ -420,7 +436,14 @@ class ScrollableBase extends Component {
 
 		this.resizeRegistry = Registry.create(this.handleResize.bind(this));
 
-		props.cbScrollTo(this.scrollTo);
+		if (props.type === 'VariableVirtualList') {
```

Comment:
Can this condition be moved to VirtualList? I think it's out of place in Scrollable which should not be concerned with the constraints of VirtualList.

Reply:
I moved it to `ScrollableVirtualList` in `VirtualListBase.js`. `ScrollableVirtualList` has a `Scrollable` and the `Scrollable` has `VirtualListBase`.